qa/L0_cmdline_trace/test.sh

```diff
@@ -295,6 +295,12 @@ set +e
 
 $TRACE_SUMMARY -t trace_ensemble.log > summary_ensemble.log
 
+if [ `grep -c "COMPUTE_INPUT_END" summary_ensemble.log` != "7" ]; then
```

Comment:
Is this a sufficient test case for the change? It might be better to actually check the reported numbers.. atleast non-zero...

Reply:
I agree that we should add more explicit check. This check can at least check if the nested traces are captured, which was removed earlier to "fix" the test.

Reply:
Ok.. Can you file a DLIS ticket for the same? We can address this in future PR.. Would be nice to get tracer working for ensemble.

Reply:
I pushed a new commit to address this. @deadeyegoodwin seems like you approved the PR right before I pushed the new changes.